opm/parser/eclipse/EclipseState/Tables/TableColumn.cpp

```diff
@@ -22,6 +22,8 @@
 #include <opm/parser/eclipse/EclipseState/Tables/ColumnSchema.hpp>
 #include <opm/parser/eclipse/EclipseState/Tables/TableColumn.hpp>
 
+#include <ert/util/ssize_t.h>
```

Comment:
what is that needed for? (in case it is `size_t`, plain c++ provides that via `#include <cstddef>`)